core/src/main/java/com/linecorp/armeria/common/logging/RequestLogProperty.java

```diff
@@ -37,16 +37,6 @@ public enum RequestLogProperty {
      */
     REQUEST_START_TIME(true),
 
-    /**
```

Comment:
Note) Understood the reason for the reorder is to ensure properties are notified in the correct order